user_validator.py

```
@@ -0,0 +1,63 @@
+    # Remove any path traversal attempts
+    filename = filename.replace('../', '')
+    filename = filename.replace('..\\', '')
+    return filename
+
+def process_user_command(user_input):
+    """
+    Process user command - SECURITY RISK: Command injection vulnerability.
+    This function directly passes user input to shell execution.
+    """
+    # WARNING: This is vulnerable to command injection
+    result = subprocess.run(
```

Ruff (0.14.13)

`subprocess` call with `shell=True` identified, security issue

(S602)